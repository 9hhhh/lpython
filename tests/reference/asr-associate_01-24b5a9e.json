{
    "basename": "asr-associate_01-24b5a9e",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/associate_01.f90",
    "infile_hash": "08d039a665a042f2a1d20a71677e2a484073202ed7421a4eb7cf011f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-associate_01-24b5a9e.stdout",
    "stdout_hash": "1d8228dd8b3f796c048a3cd159a2071d27baee2b0c2bf527642056f8",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}