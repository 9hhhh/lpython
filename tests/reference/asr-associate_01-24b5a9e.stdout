(TranslationUnit (SymbolTable 1 {associate_01: (Program (SymbolTable 2 {a: (Variable 2 a Local () Default (Real 4 [(() ())]) Source Public), b: (Variable 2 b Local () Default (Real 4 [(() ()) (() ())]) Source Public), c: (Variable 2 c Local () Default (Real 4 [(() ()) (() ()) (() ())]) Source Public), n: (Variable 2 n Local () Default (Integer 4 []) Source Public), x: (Variable 2 x Local () Default (Real 4 [(() ())]) Source Public), y: (Variable 2 y Local () Default (Real 4 [(() ()) (() ()) (() ())]) Source Public)}) associate_01 [(= (Var 2 n) (ConstantInteger 10 (Integer 4 []))) (Print () []) (Print () []) (Print () [])])}) [])
